Repro for #13000

nemanjaglumac opened the pull request:
### Status
READY

### What does this PR accomplish?
- reproduces issue #13000 

### Additional notes
The issue was already fixed, so the test should be passing.

Commit:
Add repro for #13000

flamber commented on frontend/test/metabase/scenarios/question/notebook.cy.spec.js:
```diff
@@ -129,6 +132,98 @@ describe("scenarios > question > notebook", () => {
+
+      cy.visit("/");
+      cy.findByText("Browse all items").click();
+      cy.contains("Q3").click({ force: true });
```
Looks great! But can you make sure that it can enter Notebook (Editor-button), when entering Q3?

Commit:
Improve test with added log and assertion

Commit:
Refactor the test to avoid flakiness

- it was failing in CI:
	1. finding multiple "Product IDs"
	2. XHR event "@card" never originated (locally, the card ID is 6, but in CI it seems it is 8, from what I can see in a video)
	(replacing hard coded ID `6` with a wildcard `*`)
- wrote a different flow so I don't have to remove pre-selected "Count" after I add "Sum of ..."
- added mid-point assertions on generated titles
